### PR TITLE
[upmeter] Fix D8UpmeterSmokeMiniMoreThanOnePVxPVC alert

### PR DESCRIPTION
**Pull request by haosb**
## Description

Right now, alert triggered if `smokeMini.storageClass` settings are specified in moduleconfig. It triggers if cluster have zero "bad" volumes.

PR will fix that problem:
Screenshot 2024-10-07 at 18 40 27
```
pvc-072f3721-8d29-4511-bc9a-6fae19a2558b   256Mi      RWO            Retain           Bound      d8-upmeter/disk-smoke-mini-b-0                               network-hddd            36m
pvc-6285850d-2ace-4707-9a7c-5eb244508280   256Mi      RWO            Retain           Released   d8-upmeter/disk-smoke-mini-b-0                               network-hddd            159m
pvc-6dba2a6b-0a21-4cab-b96b-12386af1a49a   256Mi      RWO            Retain           Released   d8-upmeter/disk-smoke-mini-d-0                               network-hddd            157m
pvc-829592e9-5674-499c-8205-7a323b534a91   256Mi      RWO            Retain           Bound      d8-upmeter/disk-smoke-mini-a-0                               network-hddd            159m
pvc-a35059cc-daf0-4158-a888-3c879c6963a1   256Mi      RWO            Retain           Bound      d8-upmeter/disk-smoke-mini-e-0                               network-hddd            157m
pvc-a3a072e4-8f39-4275-b70d-4372cf53ab93   256Mi      RWO            Retain           Bound      d8-upmeter/disk-smoke-mini-d-0                               network-hddd            153m
```
## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: upmeter
type: fix
summary: Fix D8UpmeterSmokeMiniMoreThanOnePVxPVC alert
impact_level: default
```

### CODE REVIEW

File: docs/documentation/_data/deckhouse-alerts.yml [hunk 1]
```diff
@@ -1907,7 +1907,7 @@ alerts:
       module: upmeter
       edition: ce
       description: |
-        PV count per smoke-mini PVC: {{ $value }}.
+        Amount of unnecessary smoke-mini PV: {{ $value }}.
```
Comment (z9r5) [inline docs/documentation/_data/deckhouse-alerts.yml:1910]:
```suggestion
        Amount of unnecessary smoke-mini PVs: {{ $value }}.
```

File: modules/500-upmeter/monitoring/prometheus-rules/alerting.yaml [hunk 2]
```diff
@@ -425,7 +424,7 @@
         plk_labels_as_annotations: "volume"
         summary: Unnecessary smoke-mini volumes in cluster
         description: |
-          PV count per smoke-mini PVC: {{ $value }}.
+          Amount of unnecessary smoke-mini PV: {{ $value }}.
```
Comment (z9r5) [inline modules/500-upmeter/monitoring/prometheus-rules/alerting.yaml:427]:
```suggestion
          Amount of unnecessary smoke-mini PVs: {{ $value }}.
```

**Review by z9r5 — APPROVED** [review]

File: docs/documentation/_data/deckhouse-alerts.yml [hunk 3]
```diff
@@ -1907,7 +1907,7 @@ alerts:
       module: upmeter
       edition: ce
       description: |
-        PV count per smoke-mini PVC: {{ $value }}.
+        Amount of unnecessary smoke-mini PVs: {{ $value }}.
```
Comment (z9r5) [inline docs/documentation/_data/deckhouse-alerts.yml:1910]:
```suggestion
        The amount of unnecessary smoke-mini PVs: {{ $value }}.
```

File: modules/500-upmeter/monitoring/prometheus-rules/alerting.yaml [hunk 4]
```diff
@@ -425,7 +424,7 @@
         plk_labels_as_annotations: "volume"
         summary: Unnecessary smoke-mini volumes in cluster
         description: |
-          PV count per smoke-mini PVC: {{ $value }}.
+          Amount of unnecessary smoke-mini PVs: {{ $value }}.
```
Comment (z9r5) [inline modules/500-upmeter/monitoring/prometheus-rules/alerting.yaml:427]:
```suggestion
          The amount of unnecessary smoke-mini PVs: {{ $value }}.
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10096](https://github.com/deckhouse/deckhouse/pull/10096) to the branch [release-1.64](https://github.com/deckhouse/deckhouse/tree/release-1.64) successful!

**Pull request merged**